python/src/opendp/_data.py

```diff
@@ -107,6 +108,38 @@ def ffislice_of_anyobjectptrs(
     return output
 
 
+@versioned
+def fill_bytes(
+    ptr,
+    len: int
+) -> bool:
+    r"""Internal function. Populate the buffer behind `ptr` with `len` random bytes
```

Comment:
Any argument for making it `_fill_bytes` if it's just internal?

Reply:
Never mind, this is generated.

Reply:
And it's in a private module!